widgetastic_patternfly.py

```diff
@@ -618,7 +618,7 @@ def select_by_visible_text(self, *items):
                 './div/ul/li/a[./span[contains(@class, "text") and normalize-space(.)={}]]'.format(
                     quote(text)),
                 parent=self)
-        self.close()
+        # self.close()
```

Comment:
@sshveta that pastebin looks like a trace from your earlier version where you were calling the fill for `select_item_type` and not using before_fill(). Can you double check that?